src/runtime/opencl/opencl_device_api.cc

```diff
@@ -208,7 +208,11 @@ std::vector<cl_device_id> GetDeviceIDs(
   cl_device_type dtype = CL_DEVICE_TYPE_ALL;
   if (device_type == "cpu") dtype = CL_DEVICE_TYPE_CPU;
   if (device_type == "gpu") dtype = CL_DEVICE_TYPE_GPU;
+#ifndef AOCL_BOARD_NAME
   if (device_type == "accelerator") dtype = CL_DEVICE_TYPE_ACCELERATOR;
+#else
+  if (device_type == "accelerator") dtype = CL_DEVICE_TYPE_DEFAULT;
```

Comment:
Why do you want to use CL_DEVICE_TYPE_DEFAULT here?

Reply:
Deleted this code and now works with device_type == "accelerator".